International relocation payments reopen (#3739)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

## How teachers can apply for the international relocation payment

> Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for teachers are open from 2 January 2024 to 29 February 2024. 

Reviewer: ```suggestion
Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for teachers are open from 2 January 2024 to 29 February 2024.
```

Author: I might be going mad @gemmadallmandfe but I can't see what's changed here?

To remain eligible for the IRP, you must apply in either the first or second term of your employment as a teacher. Applications will reopen between May and June 2024.

You will need a teaching job to apply for the IRP. [Learn more about finding a job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas).

> You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open later in 2024.

Reviewer: ```suggestion
You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications reopen later in 2024.
```

### Salaried trainees

> Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are open from 2 January 2024 to 29 February 2024. 

Reviewer: ```suggestion
Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are open from 2 January 2024 to 29 February 2024.
```

Author: Or here?

> To remain eligible for the IRP, you must apply in either the first or second term of your teacher training course. Applications will reopen between May and June 2024.  

Reviewer: ```suggestion
To remain eligible for the IRP, you must apply in either the first or second term of your teacher training course. Applications will reopen between May and June 2024.
```

> If you have not yet started your salaried teacher training course, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open in 2024.

Reviewer: ```suggestion
If you have not yet started your salaried teacher training course, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications reopen in 2024.
```
